src/cli/plugin.rs

```diff
@@ -0,0 +1,116 @@
+use crate::serve_session::ServeSession;
+
+use memofs::InMemoryFs;
+use memofs::{Vfs, VfsSnapshot};
```

Comment:
```suggestion
use memofs::{InMemoryFs, Vfs, VfsSnapshot};
```

We can merge these two lines of imports.